package.json

```diff
@@ -33,7 +33,7 @@
   "dependencies": {
     "@metamask/bip39": "^4.0.0",
     "@metamask/eth-hd-keyring": "^4.0.2",
-    "browser-passworder": "^2.0.3",
+    "browser-passworder": "MetaMask/browser-passworder#export-key",
```

Comment:
I'd recommend using a commit hash here rather than a branch. I've found in the past that referencing a branch can result in confusing situations where an older commit is still used even after the branch is updated, because Yarn has cached the resolution of this branch to the older commit. Not 100% sure whether it still works this way though.

Reply:
You're right, we do still get weird cached commit problems.  I stuck with branch as I was making frequent changes and constantly update the commit message was frustrating.

Reply:
Fair enough!